server/submission/resolvers.js

```diff
@@ -69,6 +75,55 @@ const resolvers = {
       logger.debug(`Updated Submission ${data.id} by user ${ctx.user}`)
       return newManuscript
     },
+    async finishSubmission(_, { data }, ctx) {
+      const manuscript = await db.selectId(data.id)
+      db.checkPermission(manuscript, ctx.user)
+      const newManuscript = lodash.merge({}, manuscript, data)
+
+      const { error: errorManuscript } = Joi.validate(
+        newManuscript,
+        manuscriptSchema,
+      )
+      if (errorManuscript) {
+        logger.error(`Bad manuscript data: ${errorManuscript}`)
+        throw new Error(errorManuscript)
+      }
+
+      const {
+        cosubmission,
+        cosubmissionTitle,
+        cosubmissionId,
+      } = newManuscript.submissionMeta
+      const { error: errorCosubmission } = Joi.validate(
+        {
+          cosubmission,
+          cosubmissionTitle,
+          cosubmissionId,
+        },
+        cosubmissionSchema,
+      )
+      if (errorCosubmission) {
+        logger.error(`Bad manuscript data: ${errorCosubmission}`)
+        throw new Error(errorCosubmission)
+      }
+
+      newManuscript.submissionMeta.stage = 'QA'
+      const newManuscriptDb = db.manuscriptGqlToDb(newManuscript, ctx.user)
+      await db.update(newManuscriptDb, data.id)
+
+      const mailData = {
+        from: config.get('mailer.from'),
+        to: newManuscript.submissionMeta.author.email,
```

Comment:
Should this be sent to the author or the submitter?

Reply:
What's the submitter e-mail? Right now `submissionMeta.author.email` is the e-mail from first page.

Reply:
Sorry, by submitter email I meant the email of the user submitting the submission (!). No matter though since the email is no longer required.

Reply:
So the ORCID user submitting the manuscript? Why is it no longer required?